base/src/org/compiere/process/ReplenishReport.java

```diff
@@ -696,7 +696,7 @@ private void createDO() throws Exception
 		//	For Standard Process
 		StringBuffer localWhere = new StringBuffer("AD_PInstance_ID=?");
 		if(!isMandatoryBusinessPartner) {

```

Comment:
Why does this if statement have an "!"  as in !isMandatoryBusinessParnter.  Shouldn't the test add the C_BPartner_ID > 0 when the Business Partner is mandatory?